app/models/ems_event.rb

```diff
@@ -338,6 +338,29 @@ def ems_refresh_target
     ext_management_system
   end
 
+  def src_container_replicator
```

Comment:
Done, PTAL.
- #9600 workaround TBD tomorrow in separate PR.
- Found good place to log.warn in this PR, for policy action only (other actions already log.info as they know how to handle it).